share/rpcauth/rpcauth.py

```diff
@@ -4,22 +4,20 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 from argparse import ArgumentParser
-from base64 import urlsafe_b64encode
```

Comment:
Since you're replacing `base64` lib to use `secrets`, perhaps would be good to do the same in `rpcauth-test`?